unit_tests/log/test_main_chkpt_info.cpp

```diff
@@ -960,6 +961,26 @@ mvcc_snapshot::mvcc_snapshot ()
 {
 }
 
+void
+lsa_utils::pack (const log_lsa &lsa, cubpacking::packer &serializer)
```

Comment:
Nitpicking: an alternative is to include log_lsa_utils.cpp in the project and remove duplicated functions.

Reply:
That might be better, it would have saved me some time debugging. I will change it